### PR TITLE
SAK-49636 announcementswc Add sorting options

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-49636

### CODE REVIEW

File: webcomponents/tool/src/main/frontend/packages/sakai-announcements/src/SakaiAnnouncements.js [hunk 1]
```diff
@@ -64,41 +74,67 @@ export class SakaiAnnouncements extends SakaiPageableElement {
...
-  }
+    switch (e.target.value) {
+      case TITLE_A_TO_Z:
+        this.data.sort((a1, a2) => a1.subject.localeCompare(a2.subject));
+        break;
+      case TITLE_Z_TO_A:
+        this.data.sort((a1, a2) => a2.subject.localeCompare(a1.subject));
+        break;
+      case SITE_A_TO_Z:
+        this.data.sort((a1, a2) => a1.siteTitle.localeCompare(a2.subject));
```
Comment (jesusmmp) [inline webcomponents/tool/src/main/frontend/packages/sakai-announcements/src/SakaiAnnouncements.js:87]: this.data.sort((a1, a2) => a1.siteTitle.localeCompare(a2.siteTitle));

File: webcomponents/tool/src/main/frontend/packages/sakai-announcements/src/SakaiAnnouncements.js [hunk 2]
```diff
@@ -64,41 +74,67 @@ export class SakaiAnnouncements extends SakaiPageableElement {
...
+        break;
+      case SITE_Z_TO_A:
+        this.data.sort((a1, a2) => a2.siteTitle.localeCompare(a1.subject));
```
Comment (jesusmmp) [inline webcomponents/tool/src/main/frontend/packages/sakai-announcements/src/SakaiAnnouncements.js:90]: this.data.sort((a1, a2) => a2.siteTitle.localeCompare(a1.siteTitle));

File: webcomponents/tool/src/main/frontend/packages/sakai-announcements/src/SakaiAnnouncements.js (hunk 1 above)
Comment (jesusmmp) [reply webcomponents/tool/src/main/frontend/packages/sakai-announcements/src/SakaiAnnouncements.js]: Copy & paste ;)

**Comment by adrianfish** [comment]
@jesusmmp Thanks for the review :)

**Pull request merged**